group_vars/pdc_describe/main.yml

```diff
@@ -30,3 +28,7 @@ application_host: '{{passenger_server_name}}'
 application_host_protocol: 'https'
 application_port: '443'
 project_db_host: '{{postgres_host}}'
+
+# Use Ruby 3.0.3 and install from source
+install_ruby_from_source: true
+ruby_version_override: "ruby-3.0.3"
```

Comment:
Did the name of this variable come from the fact that the old way of installing ruby was meant to "override" the default Ubuntu installed ruby? At some point in the future, I would like to see this var use the same pattern as the nodeJS var (`desired_{package_name}_version`). This will require changes across multiple projects, so it's not an immediate thing . . .